guides/source/configuring.md

```diff
@@ -1747,6 +1755,7 @@ Accepts a string for the HTML tag used to wrap attachments. Defaults to `"action
 - `config.action_dispatch.return_only_request_media_type_on_content_type`: `false`
 - `config.action_mailer.smtp_timeout`: `5`
 - `config.active_storage.video_preview_arguments`: `"-vf 'select=eq(n\\,0)+eq(key\\,1)+gt(scene\\,0.015),loop=loop=-1:size=2,trim=start_frame=1' -frames:v 1 -f image2"`
+- `config.active_storage.multiple_file_field_include_hidden`: `false`
```

Comment:
This mentions `false` as the default, but [the `7.0` defaults](https://github.com/rails/rails/pull/43511/files#diff-9f26e965e7e2a6842bd6ea755924a621038bc9dd8d63cfa6b34092aa32ede02cR239) set it to `true`.

Since this is introduced in 7.0, I might be confused by the configuring guide. What should the documentation mention as the default?

Reply:
It should be `true` in 7.0 and `false` in the base config